features/step_definitions/cms/page_template_steps.rb

```diff
@@ -12,6 +12,7 @@ def fill_draft(text)
     page.execute_script <<-JS
       $('#cms_template_draft').data('codemirror').setValue(#{text.inspect});
     JS
+    wait_for_requests
```

Comment:
So setting the data attribute `codemirror` triggers an AJAX request?

Reply:
My bad 🙈